tilecloud_chain/__init__.py

```diff
@@ -664,18 +666,33 @@ def log_tiles(tile):
             self.imap(Logger(logger, logging.INFO, '%(tilecoord)s'))
 
     def add_metatile_splitter(self):
-        store = MetaTileSplitterTileStore(
-            self.layer['mime_type'],
-            self.layer['grid_ref']['tile_size'],
-            self.layer['meta_buffer'])
+        class NullSplitter(TileStore):
+            @staticmethod
+            def get_one(tile):
+                return tile
+
+        splitters = {None: NullSplitter()}
```

Comment:
This None key is used as the default splitter if the layer name is not found in the splitters dict